### PR TITLE
RSS Feature

### PR DESCRIPTION
**Pull request by egonw**
Adds RSS feeds for author, venue, and topic. This is a full draft and ready to be tested. If working, we can discuss adding RSS icons or links somewhere on the respective author, venue, and topic pages. Just add /latest/rss after /author/Qsomething...

For example:

image

### CODE REVIEW

**Comment by fnielsen** [comment]
I am wondering whether "latest" is sufficiently telling. Could one imaging a future latest-citations? What about latest-article/rss? Is that too long?

**Comment by egonw** [comment]
It seems long... I have been thinking about this, and there is a reason why it has /latest/ ... but what other feeds to author, venue, and topic do we expect? (BTW, I think organization should have one too...)

**Comment by fnielsen** [comment]
On Google Scholar I find that the citations to my articles are perhaps the most interesting. Would that be `latest-citations` in Scholia? 

For organizations: `latest-affiliated`?

**Comment by egonw** [comment]
`latest-citations`++

But I am not sure what you mean with `latest-affiliated`?

File: scholia/rss.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+  SERVICE wikibase:label {{ bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }}
+}} ORDER BY DESC(?date)
+"""
+
+def _value(item, field):
+    return item[field]['value'] if field in item else ''
+
+def wb_get_author_latest_articles(q):
+    if not q:
+        return {}
```
Comment (fnielsen) [inline scholia/rss.py]: Why is  a dictionary return here? Shouldn't this be an empty string?

**Comment by fnielsen** [comment]
Sorry to bother you again. I changed my mind as wouldn't "work" be a better word than "articles". That will fit with the menu. 

I can change it.

**Comment by fnielsen** [comment]
I am doing it now.

File: scholia/rss.py (hunk 1 above)
Comment (egonw) [reply scholia/rss.py]: Copy/paste issue... no particular reason.

**Comment by egonw** [comment]
OK, I'll hold of hacking for now then... please ping me when done...

**Comment by fnielsen** [comment]
Running now: https://tools.wmflabs.org/scholia/venue/Q6294930/latest-works/rss

File: scholia/rss.py (hunk 1 above)
Comment (fnielsen) [reply scholia/rss.py]: I now return an empty string instead in all 3 cases.

**Pull request merged**